### ISSUE TITLE
[4.0] potential xss changelog.es6.js

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#34266]
Unsanitized input from data from a remote resource flows into innerHTML, where it is used to dynamically construct the HTML page on client side. This may result in a DOM Based Cross-Site Scripting attack

https://github.com/joomla/joomla-cms/blob/f61b9ae293e289a4a9177e6cdea12d8576f46419/build/media_source/com_installer/js/changelog.es6.js#L48

_(reported in this repo as it is unreleased code following the advice of JSST)_

**Comment by dgrammatiko** [comment]
Could use 
```js
 modal.innerHTML = Joomla.sanitizeHtml(message);
```

But, there's a catch, the `message` needs to be html not plain text (I have no clue what the message could be just by looking at the code). It could be mitigated with something like:

```js
 modal.innerHTML = Joomla.sanitizeHtml(`<div>${message}</div>`); 
```

 **IF** there's a possibility the message to be plain text)

Also this needs to be applied in all the instances of `innerHTML` were the data could be manipulated (eg server response, etc)

**Comment by brianteeman** [comment]
Closed see #34472
